Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -657,9 +654,9 @@ void takeoffClimbStage::execute(pathManager* pathMgr)
 
         takeoffRollStage::waypointStatus = takeoffRollStage::takeoffPath.get_next_directions(waypointInput, &waypointOutput);
         output = LandingTakeoffManager::translateWaypointCommands(waypointOutput);
-        
```

Comment:
Did you go through all of these files with a formatter?

Reply:
I screwed up.... My text editor by default removes all trailing white spaces. I turn it off for ZP dev but I forgot for this lol. Time to make a coding standard eih.